src/components/Content/ContentMining.vue

```diff
@@ -76,10 +88,7 @@
               class="mx--0"
               :src="require('@/assets/art/fabrication/icon.png')"
             />
-            <b>Fabrication</b> might turn it into weapons or something.
-          </span>
-          <span>
-            Or maybe they just sell it straight for
+            <b>Fabrication</b> might turn it into weapons or something, or maybe the quartermaster just sell it straight for

```

Comment:
The wording here is a little iffy, I think you're missing a `will` 

While I get that it's thematic, I worry that new players might interpret the cargo screen as being the place to sell items with this change.